std/equal.m31

```diff
@@ -427,7 +427,7 @@ let rec equality betas hints etas = handler
 
 (* TODO | as_* *)
 
-  | val ?v => fun betas hints etas => pair v (betas :: hints :: etas :: [])
+  | val ?v => fun betas hints etas => (v, betas :: hints :: etas :: [])
```

Comment:
Let's do that separately, I suspected as much but didn't want to do it. In fact, the Right Thing woudl be to have records at the level of the programming language.